pontoon/settings/base.py

```diff
@@ -164,25 +170,25 @@ def path(*args):
                 'django_jinja.builtins.extensions.DjangoFiltersExtension',
                 'pipeline.templatetags.ext.PipelineExtension',
             ],
-            'globals': {
-                'browserid_info': 'django_browserid.helpers.browserid_info',
-            }
         }
     },
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
-        'APP_DIRS': True,
+        'DIRS': [path('pontoon/templates')],
```

Comment:
Given that we have the management command, adapter and the middleware in the base app, can we also move the accounts templates there or is that not possible because of allauth not liking jinja2?